test/performance/comm/barrier/empty-chpl-barrier.chpl

```diff
@@ -0,0 +1,39 @@
+use Time;
+use Barriers;
+
+config const numTrials = 100;
+config const printTimings = false;
+
+config const numTasksPerLocale = here.maxTaskPar;
+const numTasks = numLocales * numTasksPerLocale;
+
+enum BarrierMode {
+  LocalAtomic
+};
+use BarrierMode;
+
+config param barrierMode = LocalAtomic;
+
+proc main() {
+  var t: Timer;
+
+  var LocalAtomicBarrier = new Barrier(numTasks, BarrierType.Sync);
+
+  t.start();
+  select barrierMode {
+    when LocalAtomic do LocalBarrierBarrier();
```

Comment:
This can be a TODO, but I'd slightly prefer that we use ``barrierMode`` to create the barrier, and then pass it to some common function as an argument.